xml/dm_registers.xml

```diff
@@ -580,9 +580,7 @@
             3: There was some other error (eg. alignment).
 
             4: The system bus master was busy when one of the
-            {\tt sbaddress} or {\tt sbdata} registers was written,
-            or \Rsbdatazero was read when it had stale data.
-
+            {\tt sbaddress} or {\tt sbdata} registers was written.
```

Comment:
don't you still mark error if sbdata was read when busy?

Reply:
Yes. Fixing.